  INFO root [2019-09-19 08:54:31,863] - 
  INFO root [2019-09-19 08:54:31,864] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 08:54:32,847] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 08:54:32,848] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 08:54:32,848] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 08:54:33,382] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.43.114 ( 192.168.43.114:54321 )
  INFO root [2019-09-19 08:54:33,382] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.43.114:54321, SOMAXCONN=5.
  INFO root [2019-09-19 08:54:33,383] - server-configs:
  INFO root [2019-09-19 08:54:33,383] - 	internalTcpAddr : 192.168.43.114:54321
  INFO root [2019-09-19 08:54:33,383] - 	componentID : 6000
  WARN root [2019-09-19 08:54:33,383] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 08:54:35,077] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 19088 ----
  INFO root [2019-09-19 08:54:35,077] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 08:54:35,120] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 08:54:35,142] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 08:54:35,999] - ---- cellappmgr is running ----
 DEBUG root [2019-09-19 08:54:35,999] - Components::process(): Request for the process of identity...
  WARN root [2019-09-19 08:54:39,577] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20311, errorno=10048
  INFO root [2019-09-19 08:54:43,676] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 08:54:43,815] - Components::findComponents: found logger, addr:192.168.43.114:54323
  INFO root [2019-09-19 08:54:43,815] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 08:54:43,815] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 08:54:43,817] - Components::findComponents: find dbmgr(2)...
  WARN root [2019-09-19 08:54:44,918] - Forwarding logs to logger[192.168.43.114:54323/0/0/0]...
 ERROR root [2019-09-19 08:54:53,036] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=459305740239, lastReceivedTime=459213550591, inactivityExceptionPeriod=599499873)
  INFO root [2019-09-19 08:54:53,040] - The following logs sent to logger failed:
  WARN root [2019-09-19 08:54:53,040] - ==> [2019-09-19 08:54:53 034] TCPPacketReceiver::processPendingEvents(192.168.43.114:54323): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 08:54:53,040] - ==> [2019-09-19 08:54:53 036] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=459305740239, lastReceivedTime=459213550591, inactivityExceptionPeriod=599499873)
  INFO root [2019-09-19 08:56:25,873] - 
  INFO root [2019-09-19 08:56:25,874] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 08:56:25,875] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 08:56:25,875] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 08:56:25,875] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 08:56:25,885] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.43.114 ( 192.168.43.114:54364 )
  INFO root [2019-09-19 08:56:25,886] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.43.114:54364, SOMAXCONN=5.
  INFO root [2019-09-19 08:56:25,886] - server-configs:
  INFO root [2019-09-19 08:56:25,886] - 	internalTcpAddr : 192.168.43.114:54364
  INFO root [2019-09-19 08:56:25,886] - 	componentID : 6000
  WARN root [2019-09-19 08:56:25,886] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 08:56:27,393] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 18816 ----
  INFO root [2019-09-19 08:56:27,394] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 08:56:27,394] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 08:56:27,394] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 08:56:27,769] - ---- cellappmgr is running ----
 DEBUG root [2019-09-19 08:56:27,769] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 08:56:34,895] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 08:56:34,963] - Components::findComponents: found logger, addr:192.168.43.114:54363
  INFO root [2019-09-19 08:56:34,964] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 08:56:34,964] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 08:56:34,965] - Components::findComponents: find dbmgr(2)...
  WARN root [2019-09-19 08:56:36,067] - Forwarding logs to logger[192.168.43.114:54363/0/0/0]...
  INFO root [2019-09-19 08:59:58,809] - 
  INFO root [2019-09-19 08:59:58,812] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 08:59:58,813] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 08:59:58,813] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 08:59:58,813] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 08:59:58,831] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.43.114 ( 192.168.43.114:54415 )
  INFO root [2019-09-19 08:59:58,831] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.43.114:54415, SOMAXCONN=5.
  INFO root [2019-09-19 08:59:58,831] - server-configs:
  INFO root [2019-09-19 08:59:58,831] - 	internalTcpAddr : 192.168.43.114:54415
  INFO root [2019-09-19 08:59:58,831] - 	componentID : 6000
  WARN root [2019-09-19 08:59:58,836] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 09:00:00,345] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 15476 ----
  INFO root [2019-09-19 09:00:00,346] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 09:00:00,346] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 09:00:00,347] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 09:00:00,824] - ---- cellappmgr is running ----
 DEBUG root [2019-09-19 09:00:00,825] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 09:00:08,008] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 09:00:08,048] - Components::findComponents: found logger, addr:192.168.43.114:54412
  INFO root [2019-09-19 09:00:08,048] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 09:00:08,048] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 09:00:08,050] - Components::findComponents: find dbmgr(2)...
  WARN root [2019-09-19 09:00:09,153] - Forwarding logs to logger[192.168.43.114:54412/0/0/0]...
 ERROR root [2019-09-19 09:00:11,173] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=462487116533, lastReceivedTime=462455873059, inactivityExceptionPeriod=599499873)
  INFO root [2019-09-19 09:00:11,175] - The following logs sent to logger failed:
  WARN root [2019-09-19 09:00:11,175] - ==> [2019-09-19 09:00:11 174] TCPPacketReceiver::processPendingEvents(192.168.43.114:54412): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 09:00:11,175] - ==> [2019-09-19 09:00:11 174] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=462487116533, lastReceivedTime=462455873059, inactivityExceptionPeriod=599499873)
  INFO root [2019-09-19 09:00:12,757] - 
  INFO root [2019-09-19 09:00:12,759] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 09:00:12,760] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 09:00:12,760] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 09:00:12,760] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 09:00:12,784] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.43.114 ( 192.168.43.114:54431 )
  INFO root [2019-09-19 09:00:12,784] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.43.114:54431, SOMAXCONN=5.
  INFO root [2019-09-19 09:00:12,784] - server-configs:
  INFO root [2019-09-19 09:00:12,785] - 	internalTcpAddr : 192.168.43.114:54431
  INFO root [2019-09-19 09:00:12,786] - 	componentID : 6000
  WARN root [2019-09-19 09:00:12,790] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 09:00:14,302] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 11232 ----
  INFO root [2019-09-19 09:00:14,303] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 09:00:14,303] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 09:00:14,304] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 09:00:14,689] - ---- cellappmgr is running ----
 DEBUG root [2019-09-19 09:00:14,689] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 09:00:21,825] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 09:00:21,898] - Components::findComponents: found logger, addr:192.168.43.114:54435
  INFO root [2019-09-19 09:00:21,898] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 09:00:21,898] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 09:00:21,900] - Components::findComponents: find dbmgr(2)...
  WARN root [2019-09-19 09:00:23,002] - Forwarding logs to logger[192.168.43.114:54435/0/0/0]...
 ERROR root [2019-09-19 09:00:29,498] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=462670363569, lastReceivedTime=462594378491, inactivityExceptionPeriod=599498338)
  INFO root [2019-09-19 09:00:29,499] - The following logs sent to logger failed:
  WARN root [2019-09-19 09:00:29,499] - ==> [2019-09-19 09:00:29 498] TCPPacketReceiver::processPendingEvents(192.168.43.114:54435): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 09:00:29,499] - ==> [2019-09-19 09:00:29 499] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=462670363569, lastReceivedTime=462594378491, inactivityExceptionPeriod=599498338)
  INFO root [2019-09-19 09:01:04,506] - 
  INFO root [2019-09-19 09:01:04,507] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 09:01:04,508] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 09:01:04,508] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 09:01:04,508] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 09:01:04,521] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.43.114 ( 192.168.43.114:54447 )
  INFO root [2019-09-19 09:01:04,521] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.43.114:54447, SOMAXCONN=5.
  INFO root [2019-09-19 09:01:04,521] - server-configs:
  INFO root [2019-09-19 09:01:04,521] - 	internalTcpAddr : 192.168.43.114:54447
  INFO root [2019-09-19 09:01:04,521] - 	componentID : 6000
  WARN root [2019-09-19 09:01:04,522] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  WARN root [2019-09-19 09:01:04,920] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20244, errorno=10048
  INFO root [2019-09-19 09:01:04,923] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 17456 ----
  INFO root [2019-09-19 09:01:04,925] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 09:01:04,925] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 09:01:04,925] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 09:01:05,312] - ---- cellappmgr is running ----
 DEBUG root [2019-09-19 09:01:05,313] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 09:01:12,446] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 09:01:12,521] - Components::findComponents: found logger, addr:192.168.43.114:54452
  INFO root [2019-09-19 09:01:12,521] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 09:01:12,522] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 09:01:12,523] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 09:01:12,625] - Components::findComponents: found dbmgr, addr:192.168.43.114:54446
  INFO root [2019-09-19 09:01:12,625] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 09:01:12,625] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 09:01:12,625] - Components::findComponents: found dbmgr, addr:192.168.43.114:54446
  WARN root [2019-09-19 09:01:13,627] - Forwarding logs to logger[192.168.43.114:54452/0/0/0]...
 ERROR root [2019-09-19 09:22:57,374] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=476149130814, lastReceivedTime=475921481513, inactivityExceptionPeriod=599499873)
  INFO root [2019-09-19 09:22:57,379] - The following logs sent to logger failed:
  WARN root [2019-09-19 09:22:57,379] - ==> [2019-09-19 09:22:57 371] TCPPacketReceiver::processPendingEvents(192.168.43.114:54452): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 09:22:57,379] - ==> [2019-09-19 09:22:57 376] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=476149130814, lastReceivedTime=475921481513, inactivityExceptionPeriod=599499873)
  WARN root [2019-09-19 09:22:57,449] - TCPPacketReceiver::processPendingEvents(192.168.43.114:54446): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 09:22:57,451] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=476149892628, lastReceivedTime=475920325866, inactivityExceptionPeriod=599499873)
  INFO root [2019-09-19 09:22:57,481] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 09:22:57,482] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 09:22:57,482] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 09:22:57,486] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 09:23:58,852] - 
  INFO root [2019-09-19 09:23:58,853] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 09:23:58,854] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 09:23:58,854] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 09:23:58,854] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 09:23:58,864] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.43.114 ( 192.168.43.114:55119 )
  INFO root [2019-09-19 09:23:58,864] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.43.114:55119, SOMAXCONN=5.
  INFO root [2019-09-19 09:23:58,865] - server-configs:
  INFO root [2019-09-19 09:23:58,865] - 	internalTcpAddr : 192.168.43.114:55119
  INFO root [2019-09-19 09:23:58,865] - 	componentID : 6000
  WARN root [2019-09-19 09:23:58,865] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 09:24:00,369] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 9028 ----
  INFO root [2019-09-19 09:24:00,370] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 09:24:00,370] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 09:24:00,370] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 09:24:00,774] - ---- cellappmgr is running ----
 DEBUG root [2019-09-19 09:24:00,803] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 09:24:07,952] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 09:24:07,995] - Components::findComponents: found logger, addr:192.168.43.114:55122
  INFO root [2019-09-19 09:24:07,996] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 09:24:07,996] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 09:24:07,997] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 09:24:08,099] - Components::findComponents: found dbmgr, addr:192.168.43.114:55123
  INFO root [2019-09-19 09:24:08,100] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 09:24:08,100] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 09:24:08,100] - Components::findComponents: found dbmgr, addr:192.168.43.114:55123
  WARN root [2019-09-19 09:24:09,100] - Forwarding logs to logger[192.168.43.114:55122/0/0/0]...
 ERROR root [2019-09-19 09:33:13,985] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=482315224603, lastReceivedTime=482165027781, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-19 09:33:13,987] - The following logs sent to logger failed:
  WARN root [2019-09-19 09:33:13,987] - ==> [2019-09-19 09:33:13 972] TCPPacketReceiver::processPendingEvents(192.168.43.114:55122): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 09:33:13,987] - ==> [2019-09-19 09:33:13 985] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=482315224603, lastReceivedTime=482165027781, inactivityExceptionPeriod=599500000)
  WARN root [2019-09-19 09:33:14,102] - TCPPacketReceiver::processPendingEvents(192.168.43.114:55123): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 09:33:14,107] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=482316458766, lastReceivedTime=482165037182, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-19 09:33:14,133] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 09:33:14,136] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 09:33:14,137] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 09:33:14,137] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-09-19 09:33:14,459] - ThreadPool::destroy(): waiting for thread(2)[0x15364c8,0x1536a10,], try=1
  INFO root [2019-09-19 19:05:50,691] - 
  INFO root [2019-09-19 19:05:51,312] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 19:05:52,062] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 19:05:52,062] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 19:05:52,062] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 19:05:52,171] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:60619 )
  INFO root [2019-09-19 19:05:52,171] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:60619, SOMAXCONN=5.
  INFO root [2019-09-19 19:05:52,171] - server-configs:
  INFO root [2019-09-19 19:05:52,171] - 	internalTcpAddr : 192.168.202.54:60619
  INFO root [2019-09-19 19:05:52,171] - 	componentID : 6000
  WARN root [2019-09-19 19:05:52,171] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 19:05:53,842] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 13888 ----
  INFO root [2019-09-19 19:05:53,842] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 19:05:53,872] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 19:05:53,872] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 19:05:54,622] - ---- cellappmgr is running ----
 DEBUG root [2019-09-19 19:05:54,631] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 19:06:01,892] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 19:06:01,911] - Components::findComponents: found logger, addr:192.168.202.54:60621
  INFO root [2019-09-19 19:06:01,922] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:06:01,922] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:06:01,922] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 19:06:02,032] - Components::findComponents: found dbmgr, addr:192.168.202.54:60625
  INFO root [2019-09-19 19:06:02,032] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:06:02,032] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:06:02,032] - Components::findComponents: found dbmgr, addr:192.168.202.54:60625
  WARN root [2019-09-19 19:06:03,052] - Forwarding logs to logger[192.168.202.54:60621/0/0/0]...
 ERROR root [2019-09-19 19:06:32,552] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=826394373394, lastReceivedTime=826298861384, inactivityExceptionPeriod=599500064)
  INFO root [2019-09-19 19:06:32,552] - The following logs sent to logger failed:
  WARN root [2019-09-19 19:06:32,621] - ==> [2019-09-19 19:06:32 551] TCPPacketReceiver::processPendingEvents(192.168.202.54:60621): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:06:32,621] - ==> [2019-09-19 19:06:32 554] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=826394373394, lastReceivedTime=826298861384, inactivityExceptionPeriod=599500064)
  WARN root [2019-09-19 19:06:32,621] - TCPPacketReceiver::processPendingEvents(192.168.202.54:60625): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:06:32,632] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=826395192841, lastReceivedTime=826307460809, inactivityExceptionPeriod=599500064)
  INFO root [2019-09-19 19:06:32,652] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 19:06:32,652] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 19:06:32,652] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 19:06:32,657] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 19:10:14,777] - 
  INFO root [2019-09-19 19:10:14,777] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 19:10:14,779] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 19:10:14,779] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 19:10:14,779] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 19:10:14,789] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:60730 )
  INFO root [2019-09-19 19:10:14,790] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:60730, SOMAXCONN=5.
  INFO root [2019-09-19 19:10:14,790] - server-configs:
  INFO root [2019-09-19 19:10:14,790] - 	internalTcpAddr : 192.168.202.54:60730
  INFO root [2019-09-19 19:10:14,790] - 	componentID : 6000
  WARN root [2019-09-19 19:10:14,792] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 19:10:16,297] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 1956 ----
  INFO root [2019-09-19 19:10:16,298] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 19:10:16,298] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 19:10:16,298] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 19:10:16,704] - ---- cellappmgr is running ----
 DEBUG root [2019-09-19 19:10:16,729] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 19:10:23,859] - Components::findComponents: find logger(1)...
  WARN root [2019-09-19 19:10:24,192] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20086, errorno=10048
  INFO root [2019-09-19 19:10:25,200] - Components::findComponents: find logger(2)...
  INFO root [2019-09-19 19:10:25,215] - Components::findComponents: found logger, addr:192.168.202.54:60735
  INFO root [2019-09-19 19:10:25,215] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:10:25,215] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:10:25,216] - Components::findComponents: find dbmgr(3)...
  INFO root [2019-09-19 19:10:25,319] - Components::findComponents: found dbmgr, addr:192.168.202.54:60737
  INFO root [2019-09-19 19:10:25,320] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:10:25,320] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:10:25,320] - Components::findComponents: found dbmgr, addr:192.168.202.54:60737
  WARN root [2019-09-19 19:10:26,321] - Forwarding logs to logger[192.168.202.54:60735/0/0/0]...
 ERROR root [2019-09-19 19:12:06,430] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=829733137019, lastReceivedTime=829517754781, inactivityExceptionPeriod=599500064)
  INFO root [2019-09-19 19:12:06,436] - The following logs sent to logger failed:
  WARN root [2019-09-19 19:12:06,436] - ==> [2019-09-19 19:12:06 428] TCPPacketReceiver::processPendingEvents(192.168.202.54:60735): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:12:06,437] - ==> [2019-09-19 19:12:06 431] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=829733137019, lastReceivedTime=829517754781, inactivityExceptionPeriod=599500064)
  WARN root [2019-09-19 19:12:06,561] - TCPPacketReceiver::processPendingEvents(192.168.202.54:60737): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:12:06,562] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=829734458695, lastReceivedTime=829518413587, inactivityExceptionPeriod=599500064)
  INFO root [2019-09-19 19:12:06,572] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 19:12:06,595] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 19:12:06,597] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 19:12:06,598] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 19:14:11,495] - 
  INFO root [2019-09-19 19:14:11,506] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 19:14:11,507] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 19:14:11,507] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 19:14:11,507] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 19:14:11,526] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:60805 )
  INFO root [2019-09-19 19:14:11,527] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:60805, SOMAXCONN=5.
  INFO root [2019-09-19 19:14:11,527] - server-configs:
  INFO root [2019-09-19 19:14:11,527] - 	internalTcpAddr : 192.168.202.54:60805
  INFO root [2019-09-19 19:14:11,527] - 	componentID : 6000
  WARN root [2019-09-19 19:14:11,529] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 19:14:11,928] - Components::findLogger: found logger, addr:192.168.202.54:60804
  INFO root [2019-09-19 19:14:11,928] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:14:11,928] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:14:11,934] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 15384 ----
  INFO root [2019-09-19 19:14:11,935] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 19:14:11,935] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 19:14:11,935] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 19:14:12,357] - ---- cellappmgr is running ----
 DEBUG root [2019-09-19 19:14:12,358] - Components::process(): Request for the process of identity...
  WARN root [2019-09-19 19:14:18,475] - Forwarding logs to logger[192.168.202.54:60804/0/0/0]...
 ERROR root [2019-09-19 19:15:48,604] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=831954876337, lastReceivedTime=831883890422, inactivityExceptionPeriod=599499873)
  INFO root [2019-09-19 19:15:48,607] - The following logs sent to logger failed:
  WARN root [2019-09-19 19:15:48,607] - ==> [2019-09-19 19:15:48 603] TCPPacketReceiver::processPendingEvents(192.168.202.54:60804): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:15:48,608] - ==> [2019-09-19 19:15:48 605] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=831954876337, lastReceivedTime=831883890422, inactivityExceptionPeriod=599499873)
  WARN root [2019-09-19 19:15:48,694] - TCPPacketReceiver::processPendingEvents(192.168.202.54:60800): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:15:48,696] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=831955787707, lastReceivedTime=831882842823, inactivityExceptionPeriod=599499873)
  INFO root [2019-09-19 19:15:48,725] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 19:15:48,728] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 19:15:48,728] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 19:15:48,729] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 19:16:41,348] - 
  INFO root [2019-09-19 19:16:41,349] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 19:16:41,350] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 19:16:41,350] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 19:16:41,350] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 19:16:41,360] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:60889 )
  INFO root [2019-09-19 19:16:41,360] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:60889, SOMAXCONN=5.
  INFO root [2019-09-19 19:16:41,360] - server-configs:
  INFO root [2019-09-19 19:16:41,361] - 	internalTcpAddr : 192.168.202.54:60889
  INFO root [2019-09-19 19:16:41,361] - 	componentID : 6000
  WARN root [2019-09-19 19:16:41,361] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 19:16:42,867] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 3128 ----
  INFO root [2019-09-19 19:16:42,868] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 19:16:42,868] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 19:16:42,869] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 19:16:43,285] - ---- cellappmgr is running ----
 DEBUG root [2019-09-19 19:16:43,286] - Components::process(): Request for the process of identity...
  WARN root [2019-09-19 19:16:43,624] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20931, errorno=10048
  WARN root [2019-09-19 19:16:49,772] - TCPPacketReceiver::processPendingEvents(192.168.202.54:60905): Throwing REASON_GENERAL_NETWORK - WSAECONNABORTED
  INFO root [2019-09-19 19:16:50,789] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 19:16:50,850] - Components::findComponents: found logger, addr:192.168.202.54:60893
  INFO root [2019-09-19 19:16:50,850] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:16:50,850] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:16:50,852] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 19:16:50,956] - Components::findComponents: found dbmgr, addr:192.168.202.54:60886
  INFO root [2019-09-19 19:16:50,956] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:16:50,956] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:16:50,956] - Components::findComponents: found dbmgr, addr:192.168.202.54:60886
  WARN root [2019-09-19 19:16:51,957] - Forwarding logs to logger[192.168.202.54:60893/0/0/0]...
 ERROR root [2019-09-19 19:18:07,119] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=833340034472, lastReceivedTime=833082990170, inactivityExceptionPeriod=599500320)
  INFO root [2019-09-19 19:18:07,122] - The following logs sent to logger failed:
  WARN root [2019-09-19 19:18:07,123] - ==> [2019-09-19 19:18:07 115] TCPPacketReceiver::processPendingEvents(192.168.202.54:60893): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:18:07,123] - ==> [2019-09-19 19:18:07 121] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=833340034472, lastReceivedTime=833082990170, inactivityExceptionPeriod=599500320)
  WARN root [2019-09-19 19:18:07,232] - TCPPacketReceiver::processPendingEvents(192.168.202.54:60886): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:18:07,238] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=833341214391, lastReceivedTime=833082035014, inactivityExceptionPeriod=599500320)
  INFO root [2019-09-19 19:18:07,269] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 19:18:07,275] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 19:18:07,275] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 19:18:07,276] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 19:18:33,714] - 
  INFO root [2019-09-19 19:18:33,725] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 19:18:33,726] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 19:18:33,726] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 19:18:33,726] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 19:18:33,779] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:60963 )
  INFO root [2019-09-19 19:18:33,779] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:60963, SOMAXCONN=5.
  INFO root [2019-09-19 19:18:33,779] - server-configs:
  INFO root [2019-09-19 19:18:33,779] - 	internalTcpAddr : 192.168.202.54:60963
  INFO root [2019-09-19 19:18:33,779] - 	componentID : 6000
  WARN root [2019-09-19 19:18:33,780] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  WARN root [2019-09-19 19:18:34,207] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20330, errorno=10048
  INFO root [2019-09-19 19:18:34,211] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 11052 ----
  INFO root [2019-09-19 19:18:34,211] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 19:18:34,212] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 19:18:34,212] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 19:18:34,694] - ---- cellappmgr is running ----
 DEBUG root [2019-09-19 19:18:34,716] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 19:18:41,894] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 19:18:41,939] - Components::findComponents: found logger, addr:192.168.202.54:60958
  INFO root [2019-09-19 19:18:41,939] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:18:41,939] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:18:41,941] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 19:18:42,043] - Components::findComponents: found dbmgr, addr:192.168.202.54:60960
  INFO root [2019-09-19 19:18:42,044] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:18:42,044] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:18:42,044] - Components::findComponents: found dbmgr, addr:192.168.202.54:60960
  WARN root [2019-09-19 19:18:43,044] - Forwarding logs to logger[192.168.202.54:60958/0/0/0]...
 ERROR root [2019-09-19 19:21:41,845] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=835487290619, lastReceivedTime=835404284917, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-19 19:21:41,871] - The following logs sent to logger failed:
  WARN root [2019-09-19 19:21:41,871] - ==> [2019-09-19 19:21:41 831] TCPPacketReceiver::processPendingEvents(192.168.202.54:60958): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:21:41,871] - ==> [2019-09-19 19:21:41 847] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=835487290619, lastReceivedTime=835404284917, inactivityExceptionPeriod=599500000)
  WARN root [2019-09-19 19:21:42,053] - TCPPacketReceiver::processPendingEvents(192.168.202.54:60960): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:21:42,067] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=835489505911, lastReceivedTime=835405224784, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-19 19:21:42,147] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 19:21:42,161] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 19:21:42,162] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 19:21:42,164] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-09-19 19:21:42,486] - ThreadPool::destroy(): waiting for thread(2)[0x1862b88,0x1862b20,], try=1
  INFO root [2019-09-19 19:26:03,670] - 
  INFO root [2019-09-19 19:26:03,671] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 19:26:03,673] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 19:26:03,673] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 19:26:03,673] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 19:26:03,687] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:61043 )
  INFO root [2019-09-19 19:26:03,687] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:61043, SOMAXCONN=5.
  INFO root [2019-09-19 19:26:03,687] - server-configs:
  INFO root [2019-09-19 19:26:03,688] - 	internalTcpAddr : 192.168.202.54:61043
  INFO root [2019-09-19 19:26:03,688] - 	componentID : 6000
  WARN root [2019-09-19 19:26:03,688] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 19:26:05,195] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 12524 ----
  INFO root [2019-09-19 19:26:05,195] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 19:26:05,195] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 19:26:05,196] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 19:26:05,656] - ---- cellappmgr is running ----
 DEBUG root [2019-09-19 19:26:05,676] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 19:26:12,977] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 19:26:13,068] - Components::findComponents: found logger, addr:192.168.202.54:61045
  INFO root [2019-09-19 19:26:13,069] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:26:13,069] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:26:13,071] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 19:26:13,178] - Components::findComponents: found dbmgr, addr:192.168.202.54:61047
  INFO root [2019-09-19 19:26:13,179] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:26:13,180] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:26:13,182] - Components::findComponents: found dbmgr, addr:192.168.202.54:61047
  WARN root [2019-09-19 19:26:14,184] - Forwarding logs to logger[192.168.202.54:61045/0/0/0]...
 ERROR root [2019-09-19 19:27:52,862] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=839197461800, lastReceivedTime=839005906856, inactivityExceptionPeriod=599500448)
  INFO root [2019-09-19 19:27:52,867] - The following logs sent to logger failed:
  WARN root [2019-09-19 19:27:52,868] - ==> [2019-09-19 19:27:52 862] TCPPacketReceiver::processPendingEvents(192.168.202.54:61045): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:27:52,868] - ==> [2019-09-19 19:27:52 863] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=839197461800, lastReceivedTime=839005906856, inactivityExceptionPeriod=599500448)
  WARN root [2019-09-19 19:27:52,967] - TCPPacketReceiver::processPendingEvents(192.168.202.54:61047): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:27:52,968] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=839198514704, lastReceivedTime=839008024508, inactivityExceptionPeriod=599500448)
  INFO root [2019-09-19 19:27:52,973] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 19:27:52,973] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 19:27:52,974] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 19:27:52,974] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 19:29:35,793] - 
  INFO root [2019-09-19 19:29:35,807] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 19:29:35,808] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 19:29:35,808] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 19:29:35,808] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 19:29:35,850] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:61124 )
  INFO root [2019-09-19 19:29:35,851] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:61124, SOMAXCONN=5.
  INFO root [2019-09-19 19:29:35,856] - server-configs:
  INFO root [2019-09-19 19:29:35,856] - 	internalTcpAddr : 192.168.202.54:61124
  INFO root [2019-09-19 19:29:35,856] - 	componentID : 6000
  WARN root [2019-09-19 19:29:35,862] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 19:29:37,380] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 12076 ----
  INFO root [2019-09-19 19:29:37,381] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 19:29:37,381] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 19:29:37,382] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 19:29:37,943] - ---- cellappmgr is running ----
 DEBUG root [2019-09-19 19:29:37,947] - Components::process(): Request for the process of identity...
  WARN root [2019-09-19 19:29:44,117] - TCPPacketReceiver::processPendingEvents(192.168.202.54:61142): Throwing REASON_GENERAL_NETWORK - WSAECONNABORTED
  INFO root [2019-09-19 19:29:45,122] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 19:29:45,202] - Components::findComponents: found logger, addr:192.168.202.54:61123
  INFO root [2019-09-19 19:29:45,202] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:29:45,202] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:29:45,204] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 19:29:45,311] - Components::findComponents: found dbmgr, addr:192.168.202.54:61125
  INFO root [2019-09-19 19:29:45,312] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:29:45,312] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:29:45,319] - Components::findComponents: found dbmgr, addr:192.168.202.54:61125
  WARN root [2019-09-19 19:29:46,321] - Forwarding logs to logger[192.168.202.54:61123/0/0/0]...
 ERROR root [2019-09-19 19:31:10,678] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=841175615453, lastReceivedTime=841126932542, inactivityExceptionPeriod=599499169)
  INFO root [2019-09-19 19:31:10,695] - The following logs sent to logger failed:
  WARN root [2019-09-19 19:31:10,696] - ==> [2019-09-19 19:31:10 676] TCPPacketReceiver::processPendingEvents(192.168.202.54:61123): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:31:10,696] - ==> [2019-09-19 19:31:10 679] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=841175615453, lastReceivedTime=841126932542, inactivityExceptionPeriod=599499169)
  WARN root [2019-09-19 19:31:10,795] - TCPPacketReceiver::processPendingEvents(192.168.202.54:61125): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:31:10,795] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=841176784185, lastReceivedTime=841127401359, inactivityExceptionPeriod=599499169)
  INFO root [2019-09-19 19:31:10,804] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 19:31:10,806] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 19:31:10,806] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 19:31:10,806] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-09-19 19:31:11,109] - ThreadPool::destroy(): waiting for thread(2)[0x15effd8,0x15ef8f0,], try=1
  INFO root [2019-09-19 19:34:51,085] - 
  INFO root [2019-09-19 19:34:51,092] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 19:34:51,093] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 19:34:51,093] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 19:34:51,093] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 19:34:51,182] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:61218 )
  INFO root [2019-09-19 19:34:51,182] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:61218, SOMAXCONN=5.
  INFO root [2019-09-19 19:34:51,182] - server-configs:
  INFO root [2019-09-19 19:34:51,183] - 	internalTcpAddr : 192.168.202.54:61218
  INFO root [2019-09-19 19:34:51,183] - 	componentID : 6000
  WARN root [2019-09-19 19:34:51,193] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 19:34:52,697] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 5192 ----
  INFO root [2019-09-19 19:34:52,698] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 19:34:52,698] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 19:34:52,698] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 19:34:53,237] - ---- cellappmgr is running ----
 DEBUG root [2019-09-19 19:34:53,258] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 19:35:00,473] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 19:35:00,530] - Components::findComponents: found logger, addr:192.168.202.54:61216
  INFO root [2019-09-19 19:35:00,530] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:35:00,530] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:35:00,534] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 19:35:00,639] - Components::findComponents: found dbmgr, addr:192.168.202.54:61214
  INFO root [2019-09-19 19:35:00,639] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:35:00,639] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:35:00,660] - Components::findComponents: found dbmgr, addr:192.168.202.54:61214
  WARN root [2019-09-19 19:35:01,661] - Forwarding logs to logger[192.168.202.54:61216/0/0/0]...
 ERROR root [2019-09-19 19:36:06,970] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=844138539806, lastReceivedTime=843979627922, inactivityExceptionPeriod=599501024)
  INFO root [2019-09-19 19:36:06,973] - The following logs sent to logger failed:
  WARN root [2019-09-19 19:36:06,974] - ==> [2019-09-19 19:36:06 964] TCPPacketReceiver::processPendingEvents(192.168.202.54:61216): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:36:06,974] - ==> [2019-09-19 19:36:06 971] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=844138539806, lastReceivedTime=843979627922, inactivityExceptionPeriod=599501024)
  WARN root [2019-09-19 19:36:07,068] - TCPPacketReceiver::processPendingEvents(192.168.202.54:61214): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:36:07,083] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=844139662603, lastReceivedTime=843979324346, inactivityExceptionPeriod=599501024)
  INFO root [2019-09-19 19:36:07,114] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 19:36:07,116] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 19:36:07,117] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 19:36:07,117] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 19:38:55,526] - 
  INFO root [2019-09-19 19:38:55,531] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 19:38:55,532] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 19:38:55,532] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 19:38:55,533] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 19:38:55,574] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:61302 )
  INFO root [2019-09-19 19:38:55,574] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:61302, SOMAXCONN=5.
  INFO root [2019-09-19 19:38:55,575] - server-configs:
  INFO root [2019-09-19 19:38:55,575] - 	internalTcpAddr : 192.168.202.54:61302
  INFO root [2019-09-19 19:38:55,575] - 	componentID : 6000
  WARN root [2019-09-19 19:38:55,580] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 19:38:57,098] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 8208 ----
  INFO root [2019-09-19 19:38:57,099] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 19:38:57,099] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 19:38:57,100] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 19:38:57,494] - ---- cellappmgr is running ----
 DEBUG root [2019-09-19 19:38:57,495] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 19:39:04,621] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 19:39:04,697] - Components::findComponents: found logger, addr:192.168.202.54:61304
  INFO root [2019-09-19 19:39:04,697] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:39:04,697] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:39:04,699] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 19:39:04,801] - Components::findComponents: found dbmgr, addr:192.168.202.54:61299
  INFO root [2019-09-19 19:39:04,802] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:39:04,802] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:39:04,802] - Components::findComponents: found dbmgr, addr:192.168.202.54:61299
  WARN root [2019-09-19 19:39:05,802] - Forwarding logs to logger[192.168.202.54:61304/0/0/0]...
 ERROR root [2019-09-19 19:42:02,098] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=847689810723, lastReceivedTime=847624762347, inactivityExceptionPeriod=599498785)
  INFO root [2019-09-19 19:42:02,100] - The following logs sent to logger failed:
  WARN root [2019-09-19 19:42:02,100] - ==> [2019-09-19 19:42:02 095] TCPPacketReceiver::processPendingEvents(192.168.202.54:61304): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:42:02,100] - ==> [2019-09-19 19:42:02 098] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=847689810723, lastReceivedTime=847624762347, inactivityExceptionPeriod=599498785)
  WARN root [2019-09-19 19:42:02,207] - TCPPacketReceiver::processPendingEvents(192.168.202.54:61299): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:42:02,209] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=847690932512, lastReceivedTime=847624247438, inactivityExceptionPeriod=599498785)
  INFO root [2019-09-19 19:42:02,241] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 19:42:02,243] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 19:42:02,243] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 19:42:02,243] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 19:43:42,205] - 
  INFO root [2019-09-19 19:43:42,211] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 19:43:42,212] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 19:43:42,212] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 19:43:42,212] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 19:43:42,231] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:61377 )
  INFO root [2019-09-19 19:43:42,232] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:61377, SOMAXCONN=5.
  INFO root [2019-09-19 19:43:42,232] - server-configs:
  INFO root [2019-09-19 19:43:42,232] - 	internalTcpAddr : 192.168.202.54:61377
  INFO root [2019-09-19 19:43:42,232] - 	componentID : 6000
  WARN root [2019-09-19 19:43:42,236] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 19:43:43,741] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 19280 ----
  INFO root [2019-09-19 19:43:43,741] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 19:43:43,742] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 19:43:43,742] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 19:43:44,176] - ---- cellappmgr is running ----
 DEBUG root [2019-09-19 19:43:44,177] - Components::process(): Request for the process of identity...
  WARN root [2019-09-19 19:43:47,620] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20584, errorno=10048
  WARN root [2019-09-19 19:43:50,664] - TCPPacketReceiver::processPendingEvents(192.168.202.54:61393): Throwing REASON_GENERAL_NETWORK - WSAECONNABORTED
  INFO root [2019-09-19 19:43:51,665] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 19:43:51,692] - Components::findComponents: found logger, addr:192.168.202.54:61374
  INFO root [2019-09-19 19:43:51,692] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:43:51,692] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:43:51,694] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 19:43:51,796] - Components::findComponents: found dbmgr, addr:192.168.202.54:61372
  INFO root [2019-09-19 19:43:51,796] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:43:51,797] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:43:51,797] - Components::findComponents: found dbmgr, addr:192.168.202.54:61372
  WARN root [2019-09-19 19:43:52,798] - Forwarding logs to logger[192.168.202.54:61374/0/0/0]...
 ERROR root [2019-09-19 19:47:40,858] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=851077425761, lastReceivedTime=850790133915, inactivityExceptionPeriod=599500320)
  INFO root [2019-09-19 19:47:40,863] - The following logs sent to logger failed:
  WARN root [2019-09-19 19:47:40,863] - ==> [2019-09-19 19:47:40 858] TCPPacketReceiver::processPendingEvents(192.168.202.54:61374): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:47:40,864] - ==> [2019-09-19 19:47:40 860] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=851077425761, lastReceivedTime=850790133915, inactivityExceptionPeriod=599500320)
  WARN root [2019-09-19 19:47:40,944] - TCPPacketReceiver::processPendingEvents(192.168.202.54:61372): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:47:40,945] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=851078285566, lastReceivedTime=850789964923, inactivityExceptionPeriod=599500320)
  INFO root [2019-09-19 19:47:40,949] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 19:47:40,949] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 19:47:40,950] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 19:47:40,950] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 19:47:42,658] - 
  INFO root [2019-09-19 19:47:42,679] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 19:47:42,680] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 19:47:42,680] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 19:47:42,681] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 19:47:42,703] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:61456 )
  INFO root [2019-09-19 19:47:42,704] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:61456, SOMAXCONN=5.
  INFO root [2019-09-19 19:47:42,704] - server-configs:
  INFO root [2019-09-19 19:47:42,704] - 	internalTcpAddr : 192.168.202.54:61456
  INFO root [2019-09-19 19:47:42,704] - 	componentID : 6000
  WARN root [2019-09-19 19:47:42,705] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  WARN root [2019-09-19 19:47:43,048] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20941, errorno=10048
  INFO root [2019-09-19 19:47:43,052] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 14240 ----
  INFO root [2019-09-19 19:47:43,052] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 19:47:43,052] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 19:47:43,053] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 19:47:43,444] - ---- cellappmgr is running ----
 DEBUG root [2019-09-19 19:47:43,445] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 19:47:50,555] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 19:47:50,654] - Components::findComponents: found logger, addr:192.168.202.54:61455
  INFO root [2019-09-19 19:47:50,654] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:47:50,654] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:47:50,655] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 19:47:50,757] - Components::findComponents: found dbmgr, addr:192.168.202.54:61457
  INFO root [2019-09-19 19:47:50,757] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:47:50,758] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:47:50,758] - Components::findComponents: found dbmgr, addr:192.168.202.54:61457
  WARN root [2019-09-19 19:47:51,759] - Forwarding logs to logger[192.168.202.54:61455/0/0/0]...
 ERROR root [2019-09-19 19:54:45,049] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=855319334039, lastReceivedTime=855295566637, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-19 19:54:45,058] - The following logs sent to logger failed:
  WARN root [2019-09-19 19:54:45,058] - ==> [2019-09-19 19:54:45 048] TCPPacketReceiver::processPendingEvents(192.168.202.54:61455): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:54:45,059] - ==> [2019-09-19 19:54:45 050] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=855319334039, lastReceivedTime=855295566637, inactivityExceptionPeriod=599500000)
  WARN root [2019-09-19 19:54:45,132] - TCPPacketReceiver::processPendingEvents(192.168.202.54:61457): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:54:45,135] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=855320186252, lastReceivedTime=855295327349, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-19 19:54:45,143] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 19:54:45,144] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 19:54:45,145] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 19:54:45,145] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 19:54:46,725] - 
  INFO root [2019-09-19 19:54:46,726] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 19:54:46,727] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 19:54:46,728] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 19:54:46,728] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 19:54:46,757] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:61526 )
  INFO root [2019-09-19 19:54:46,757] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:61526, SOMAXCONN=5.
  INFO root [2019-09-19 19:54:46,757] - server-configs:
  INFO root [2019-09-19 19:54:46,757] - 	internalTcpAddr : 192.168.202.54:61526
  INFO root [2019-09-19 19:54:46,757] - 	componentID : 6000
  WARN root [2019-09-19 19:54:46,764] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  WARN root [2019-09-19 19:54:47,108] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20497, errorno=10048
  INFO root [2019-09-19 19:54:47,112] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 5292 ----
  INFO root [2019-09-19 19:54:47,112] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 19:54:47,112] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 19:54:47,113] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 19:54:47,506] - ---- cellappmgr is running ----
 DEBUG root [2019-09-19 19:54:47,506] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 19:54:54,615] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 19:54:54,673] - Components::findComponents: found logger, addr:192.168.202.54:61521
  INFO root [2019-09-19 19:54:54,674] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:54:54,674] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:54:54,675] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 19:54:54,777] - Components::findComponents: found dbmgr, addr:192.168.202.54:61520
  INFO root [2019-09-19 19:54:54,778] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:54:54,778] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:54:54,778] - Components::findComponents: found dbmgr, addr:192.168.202.54:61520
  WARN root [2019-09-19 19:54:55,780] - Forwarding logs to logger[192.168.202.54:61521/0/0/0]...
 ERROR root [2019-09-19 19:58:33,488] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=857603716809, lastReceivedTime=857435953039, inactivityExceptionPeriod=599499617)
  INFO root [2019-09-19 19:58:33,490] - The following logs sent to logger failed:
  WARN root [2019-09-19 19:58:33,490] - ==> [2019-09-19 19:58:33 485] TCPPacketReceiver::processPendingEvents(192.168.202.54:61521): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:58:33,491] - ==> [2019-09-19 19:58:33 489] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=857603716809, lastReceivedTime=857435953039, inactivityExceptionPeriod=599499617)
  WARN root [2019-09-19 19:58:33,574] - TCPPacketReceiver::processPendingEvents(192.168.202.54:61520): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:58:33,579] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=857604625712, lastReceivedTime=857436002462, inactivityExceptionPeriod=599499617)
  INFO root [2019-09-19 19:58:33,593] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 19:58:33,597] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 19:58:33,597] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 19:58:33,598] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 19:58:35,131] - 
  INFO root [2019-09-19 19:58:35,132] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 19:58:35,133] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 19:58:35,133] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 19:58:35,133] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 19:58:35,143] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:61598 )
  INFO root [2019-09-19 19:58:35,143] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:61598, SOMAXCONN=5.
  INFO root [2019-09-19 19:58:35,144] - server-configs:
  INFO root [2019-09-19 19:58:35,144] - 	internalTcpAddr : 192.168.202.54:61598
  INFO root [2019-09-19 19:58:35,144] - 	componentID : 6000
  WARN root [2019-09-19 19:58:35,144] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 19:58:36,649] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 6532 ----
  INFO root [2019-09-19 19:58:36,650] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 19:58:36,650] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 19:58:36,651] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 19:58:37,099] - ---- cellappmgr is running ----
 DEBUG root [2019-09-19 19:58:37,100] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 19:58:44,212] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 19:58:44,254] - Components::findComponents: found logger, addr:192.168.202.54:61597
  INFO root [2019-09-19 19:58:44,255] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:58:44,255] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:58:44,256] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 19:58:44,360] - Components::findComponents: found dbmgr, addr:192.168.202.54:61600
  INFO root [2019-09-19 19:58:44,361] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:58:44,362] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:58:44,362] - Components::findComponents: found dbmgr, addr:192.168.202.54:61600
  WARN root [2019-09-19 19:58:45,364] - Forwarding logs to logger[192.168.202.54:61597/0/0/0]...
 ERROR root [2019-09-19 20:01:48,647] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=859555307703, lastReceivedTime=859420104568, inactivityExceptionPeriod=599499873)
  INFO root [2019-09-19 20:01:48,664] - The following logs sent to logger failed:
  WARN root [2019-09-19 20:01:48,664] - ==> [2019-09-19 20:01:48 637] TCPPacketReceiver::processPendingEvents(192.168.202.54:61597): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:01:48,664] - ==> [2019-09-19 20:01:48 648] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=859555307703, lastReceivedTime=859420104568, inactivityExceptionPeriod=599499873)
  WARN root [2019-09-19 20:01:48,749] - TCPPacketReceiver::processPendingEvents(192.168.202.54:61600): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:01:48,756] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=859556404128, lastReceivedTime=859420791498, inactivityExceptionPeriod=599499873)
  INFO root [2019-09-19 20:01:48,801] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 20:01:48,802] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 20:01:48,804] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 20:01:48,805] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 20:01:50,845] - 
  INFO root [2019-09-19 20:01:50,849] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 20:01:50,850] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 20:01:50,851] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 20:01:50,851] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 20:01:50,980] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:61666 )
  INFO root [2019-09-19 20:01:50,981] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:61666, SOMAXCONN=5.
  INFO root [2019-09-19 20:01:50,981] - server-configs:
  INFO root [2019-09-19 20:01:50,981] - 	internalTcpAddr : 192.168.202.54:61666
  INFO root [2019-09-19 20:01:50,981] - 	componentID : 6000
  WARN root [2019-09-19 20:01:50,983] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 20:01:52,503] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 13832 ----
  INFO root [2019-09-19 20:01:52,503] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 20:01:52,504] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 20:01:52,504] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 20:01:52,976] - ---- cellappmgr is running ----
 DEBUG root [2019-09-19 20:01:52,994] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 20:02:00,166] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 20:02:00,211] - Components::findComponents: found logger, addr:192.168.202.54:61662
  INFO root [2019-09-19 20:02:00,211] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 20:02:00,211] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 20:02:00,213] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 20:02:00,316] - Components::findComponents: found dbmgr, addr:192.168.202.54:61668
  INFO root [2019-09-19 20:02:00,317] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 20:02:00,317] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 20:02:00,317] - Components::findComponents: found dbmgr, addr:192.168.202.54:61668
  WARN root [2019-09-19 20:02:01,317] - Forwarding logs to logger[192.168.202.54:61662/0/0/0]...
 ERROR root [2019-09-19 20:04:12,612] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=860994955694, lastReceivedTime=860778461418, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-19 20:04:12,617] - The following logs sent to logger failed:
  WARN root [2019-09-19 20:04:12,618] - ==> [2019-09-19 20:04:12 610] TCPPacketReceiver::processPendingEvents(192.168.202.54:61662): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:04:12,618] - ==> [2019-09-19 20:04:12 613] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=860994955694, lastReceivedTime=860778461418, inactivityExceptionPeriod=599500000)
  WARN root [2019-09-19 20:04:12,715] - TCPPacketReceiver::processPendingEvents(192.168.202.54:61668): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:04:12,718] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=860996015139, lastReceivedTime=860778967236, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-19 20:04:12,748] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 20:04:12,749] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 20:04:12,749] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 20:04:12,749] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 20:05:38,284] - 
  INFO root [2019-09-19 20:05:38,288] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 20:05:38,289] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 20:05:38,290] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 20:05:38,290] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 20:05:38,319] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:61737 )
  INFO root [2019-09-19 20:05:38,319] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:61737, SOMAXCONN=5.
  INFO root [2019-09-19 20:05:38,319] - server-configs:
  INFO root [2019-09-19 20:05:38,320] - 	internalTcpAddr : 192.168.202.54:61737
  INFO root [2019-09-19 20:05:38,320] - 	componentID : 6000
  WARN root [2019-09-19 20:05:38,320] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  WARN root [2019-09-19 20:05:38,662] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20255, errorno=10048
  INFO root [2019-09-19 20:05:38,665] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 3204 ----
  INFO root [2019-09-19 20:05:38,666] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 20:05:38,666] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 20:05:38,666] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 20:05:39,091] - ---- cellappmgr is running ----
 DEBUG root [2019-09-19 20:05:39,093] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 20:05:46,229] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 20:05:46,249] - Components::findComponents: found logger, addr:192.168.202.54:61732
  INFO root [2019-09-19 20:05:46,250] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 20:05:46,250] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 20:05:46,251] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 20:05:46,355] - Components::findComponents: found dbmgr, addr:192.168.202.54:61733
  INFO root [2019-09-19 20:05:46,356] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 20:05:46,356] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 20:05:46,358] - Components::findComponents: found dbmgr, addr:192.168.202.54:61733
  WARN root [2019-09-19 20:05:47,359] - Forwarding logs to logger[192.168.202.54:61732/0/0/0]...
 ERROR root [2019-09-19 20:08:50,262] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=863771450536, lastReceivedTime=863648758606, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-19 20:08:50,265] - The following logs sent to logger failed:
  WARN root [2019-09-19 20:08:50,266] - ==> [2019-09-19 20:08:50 259] TCPPacketReceiver::processPendingEvents(192.168.202.54:61732): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:08:50,266] - ==> [2019-09-19 20:08:50 262] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=863771450536, lastReceivedTime=863648758606, inactivityExceptionPeriod=599500000)
  WARN root [2019-09-19 20:08:50,360] - TCPPacketReceiver::processPendingEvents(192.168.202.54:61733): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:08:50,364] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=863772477844, lastReceivedTime=863650361408, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-19 20:08:50,385] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 20:08:50,392] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 20:08:50,392] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 20:08:50,392] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 20:22:58,197] - 
  INFO root [2019-09-19 20:22:58,206] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 20:22:58,207] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 20:22:58,207] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 20:22:58,208] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 20:22:58,229] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:61854 )
  INFO root [2019-09-19 20:22:58,229] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:61854, SOMAXCONN=5.
  INFO root [2019-09-19 20:22:58,229] - server-configs:
  INFO root [2019-09-19 20:22:58,229] - 	internalTcpAddr : 192.168.202.54:61854
  INFO root [2019-09-19 20:22:58,229] - 	componentID : 6000
  WARN root [2019-09-19 20:22:58,232] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  WARN root [2019-09-19 20:22:58,745] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20331, errorno=10048
  INFO root [2019-09-19 20:22:58,749] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 13892 ----
  INFO root [2019-09-19 20:22:58,749] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 20:22:58,749] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 20:22:58,750] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 20:22:59,160] - ---- cellappmgr is running ----
 DEBUG root [2019-09-19 20:22:59,161] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 20:23:06,292] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 20:23:06,376] - Components::findComponents: found logger, addr:192.168.202.54:61851
  INFO root [2019-09-19 20:23:06,376] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 20:23:06,376] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 20:23:06,377] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 20:23:06,479] - Components::findComponents: found dbmgr, addr:192.168.202.54:61855
  INFO root [2019-09-19 20:23:06,480] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 20:23:06,480] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 20:23:06,480] - Components::findComponents: found dbmgr, addr:192.168.202.54:61855
  WARN root [2019-09-19 20:23:07,481] - Forwarding logs to logger[192.168.202.54:61851/0/0/0]...
 ERROR root [2019-09-19 20:24:32,581] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=873194645324, lastReceivedTime=873150314761, inactivityExceptionPeriod=599500640)
  INFO root [2019-09-19 20:24:32,584] - The following logs sent to logger failed:
  WARN root [2019-09-19 20:24:32,585] - ==> [2019-09-19 20:24:32 578] TCPPacketReceiver::processPendingEvents(192.168.202.54:61851): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:24:32,585] - ==> [2019-09-19 20:24:32 582] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=873194645324, lastReceivedTime=873150314761, inactivityExceptionPeriod=599500640)
  WARN root [2019-09-19 20:24:32,672] - TCPPacketReceiver::processPendingEvents(192.168.202.54:61855): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:24:32,673] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=873195569528, lastReceivedTime=873151621333, inactivityExceptionPeriod=599500640)
  INFO root [2019-09-19 20:24:32,679] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 20:24:32,680] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 20:24:32,680] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 20:24:32,680] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 20:25:22,585] - 
  INFO root [2019-09-19 20:25:22,586] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 20:25:22,587] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 20:25:22,587] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 20:25:22,587] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 20:25:22,596] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:61916 )
  INFO root [2019-09-19 20:25:22,596] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:61916, SOMAXCONN=5.
  INFO root [2019-09-19 20:25:22,596] - server-configs:
  INFO root [2019-09-19 20:25:22,596] - 	internalTcpAddr : 192.168.202.54:61916
  INFO root [2019-09-19 20:25:22,597] - 	componentID : 6000
  WARN root [2019-09-19 20:25:22,597] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 20:25:22,945] - Components::findLogger: found logger, addr:192.168.202.54:61913
  INFO root [2019-09-19 20:25:22,946] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 20:25:22,946] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 20:25:22,950] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 18376 ----
  INFO root [2019-09-19 20:25:22,951] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 20:25:22,951] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 20:25:22,951] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 20:25:23,454] - ---- cellappmgr is running ----
 DEBUG root [2019-09-19 20:25:23,477] - Components::process(): Request for the process of identity...
  WARN root [2019-09-19 20:25:29,598] - Forwarding logs to logger[192.168.202.54:61913/0/0/0]...
 ERROR root [2019-09-19 20:27:30,394] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=874972783486, lastReceivedTime=874893630948, inactivityExceptionPeriod=599499873)
  INFO root [2019-09-19 20:27:30,400] - The following logs sent to logger failed:
  WARN root [2019-09-19 20:27:30,401] - ==> [2019-09-19 20:27:30 388] TCPPacketReceiver::processPendingEvents(192.168.202.54:61913): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:27:30,401] - ==> [2019-09-19 20:27:30 396] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=874972783486, lastReceivedTime=874893630948, inactivityExceptionPeriod=599499873)
  WARN root [2019-09-19 20:27:30,490] - TCPPacketReceiver::processPendingEvents(192.168.202.54:61914): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:27:30,494] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=874973781066, lastReceivedTime=874894030507, inactivityExceptionPeriod=599499873)
  INFO root [2019-09-19 20:27:30,506] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 20:27:30,508] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 20:27:30,508] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 20:27:30,509] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 20:29:07,383] - 
  INFO root [2019-09-19 20:29:07,393] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 20:29:07,394] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 20:29:07,394] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 20:29:07,394] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 20:29:07,426] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:62086 )
  INFO root [2019-09-19 20:29:07,426] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:62086, SOMAXCONN=5.
  INFO root [2019-09-19 20:29:07,426] - server-configs:
  INFO root [2019-09-19 20:29:07,427] - 	internalTcpAddr : 192.168.202.54:62086
  INFO root [2019-09-19 20:29:07,427] - 	componentID : 6000
  WARN root [2019-09-19 20:29:07,437] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 20:29:08,951] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 6316 ----
  INFO root [2019-09-19 20:29:08,951] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 20:29:08,951] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 20:29:08,952] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 20:29:09,496] - ---- cellappmgr is running ----
 DEBUG root [2019-09-19 20:29:09,500] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 20:29:16,712] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 20:29:16,774] - Components::findComponents: found logger, addr:192.168.202.54:62085
  INFO root [2019-09-19 20:29:16,774] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 20:29:16,774] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 20:29:16,775] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 20:29:16,780] - Components::findComponents: found logger, addr:192.168.202.54:62085
  INFO root [2019-09-19 20:29:16,783] - Components::findComponents: found dbmgr, addr:192.168.202.54:62082
  INFO root [2019-09-19 20:29:16,783] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 20:29:16,784] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 20:29:16,884] - Components::findComponents: found dbmgr, addr:192.168.202.54:62082
  WARN root [2019-09-19 20:29:17,885] - Forwarding logs to logger[192.168.202.54:62085/0/0/0]...
 ERROR root [2019-09-19 20:30:42,967] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=876898506721, lastReceivedTime=876843129391, inactivityExceptionPeriod=599479154)
  INFO root [2019-09-19 20:30:42,972] - The following logs sent to logger failed:
  WARN root [2019-09-19 20:30:42,972] - ==> [2019-09-19 20:30:42 963] TCPPacketReceiver::processPendingEvents(192.168.202.54:62085): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:30:42,973] - ==> [2019-09-19 20:30:42 968] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=876898506721, lastReceivedTime=876843129391, inactivityExceptionPeriod=599479154)
  WARN root [2019-09-19 20:30:43,055] - TCPPacketReceiver::processPendingEvents(192.168.202.54:62082): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:30:43,057] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=876899402121, lastReceivedTime=876842837733, inactivityExceptionPeriod=599479154)
  INFO root [2019-09-19 20:30:43,076] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 20:30:43,084] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 20:30:43,085] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 20:30:43,085] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 20:30:44,610] - 
  INFO root [2019-09-19 20:30:44,611] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 20:30:44,612] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 20:30:44,612] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 20:30:44,612] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 20:30:44,659] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:62153 )
  INFO root [2019-09-19 20:30:44,659] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:62153, SOMAXCONN=5.
  INFO root [2019-09-19 20:30:44,659] - server-configs:
  INFO root [2019-09-19 20:30:44,659] - 	internalTcpAddr : 192.168.202.54:62153
  INFO root [2019-09-19 20:30:44,660] - 	componentID : 6000
  WARN root [2019-09-19 20:30:44,661] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 20:30:46,180] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 16124 ----
  INFO root [2019-09-19 20:30:46,181] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 20:30:46,181] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 20:30:46,181] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 20:30:46,607] - ---- cellappmgr is running ----
 DEBUG root [2019-09-19 20:30:46,611] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 20:30:53,763] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 20:30:53,839] - Components::findComponents: found logger, addr:192.168.202.54:62155
  INFO root [2019-09-19 20:30:53,839] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 20:30:53,839] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 20:30:53,841] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 20:30:53,845] - Components::findComponents: found dbmgr, addr:192.168.202.54:62156
  INFO root [2019-09-19 20:30:53,845] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 20:30:53,845] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 20:30:53,946] - Components::findComponents: found dbmgr, addr:192.168.202.54:62156
  WARN root [2019-09-19 20:30:54,948] - Forwarding logs to logger[192.168.202.54:62155/0/0/0]...
 ERROR root [2019-09-19 20:32:48,171] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=878150545310, lastReceivedTime=878115583569, inactivityExceptionPeriod=599498658)
  INFO root [2019-09-19 20:32:48,179] - The following logs sent to logger failed:
  WARN root [2019-09-19 20:32:48,180] - ==> [2019-09-19 20:32:48 170] TCPPacketReceiver::processPendingEvents(192.168.202.54:62155): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:32:48,180] - ==> [2019-09-19 20:32:48 172] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=878150545310, lastReceivedTime=878115583569, inactivityExceptionPeriod=599498658)
  WARN root [2019-09-19 20:32:48,298] - TCPPacketReceiver::processPendingEvents(192.168.202.54:62156): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:32:48,299] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=878151830332, lastReceivedTime=878115469541, inactivityExceptionPeriod=599498658)
  INFO root [2019-09-19 20:32:48,333] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 20:32:48,344] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 20:32:48,346] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 20:32:48,352] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 20:32:50,239] - 
  INFO root [2019-09-19 20:32:50,240] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 20:32:50,241] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 20:32:50,242] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 20:32:50,242] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 20:32:50,252] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:62247 )
  INFO root [2019-09-19 20:32:50,252] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:62247, SOMAXCONN=5.
  INFO root [2019-09-19 20:32:50,253] - server-configs:
  INFO root [2019-09-19 20:32:50,253] - 	internalTcpAddr : 192.168.202.54:62247
  INFO root [2019-09-19 20:32:50,253] - 	componentID : 6000
  WARN root [2019-09-19 20:32:50,257] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 20:32:51,765] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 10028 ----
  INFO root [2019-09-19 20:32:51,765] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 20:32:51,765] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 20:32:51,766] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 20:32:52,236] - ---- cellappmgr is running ----
 DEBUG root [2019-09-19 20:32:52,239] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 20:32:59,444] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 20:32:59,470] - Components::findComponents: found logger, addr:192.168.202.54:62244
  INFO root [2019-09-19 20:32:59,471] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 20:32:59,471] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 20:32:59,474] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 20:32:59,475] - Components::findComponents: found logger, addr:192.168.202.54:62244
  INFO root [2019-09-19 20:32:59,478] - Components::findComponents: found dbmgr, addr:192.168.202.54:62248
  INFO root [2019-09-19 20:32:59,479] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 20:32:59,479] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 20:32:59,580] - Components::findComponents: found dbmgr, addr:192.168.202.54:62248
  WARN root [2019-09-19 20:33:00,582] - Forwarding logs to logger[192.168.202.54:62244/0/0/0]...
 ERROR root [2019-09-19 20:34:50,134] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=879370179358, lastReceivedTime=879070782618, inactivityExceptionPeriod=599498785)
  INFO root [2019-09-19 20:34:50,145] - The following logs sent to logger failed:
  WARN root [2019-09-19 20:34:50,145] - ==> [2019-09-19 20:34:50 133] TCPPacketReceiver::processPendingEvents(192.168.202.54:62244): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:34:50,146] - ==> [2019-09-19 20:34:50 135] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=879370179358, lastReceivedTime=879070782618, inactivityExceptionPeriod=599498785)
  WARN root [2019-09-19 20:34:50,239] - TCPPacketReceiver::processPendingEvents(192.168.202.54:62248): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:34:50,241] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=879371244706, lastReceivedTime=879370660909, inactivityExceptionPeriod=599498785)
  INFO root [2019-09-19 20:34:50,258] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 20:34:50,264] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 20:34:50,264] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 20:34:50,264] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 20:40:30,681] - 
  INFO root [2019-09-19 20:40:30,682] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 20:40:30,682] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 20:40:30,683] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 20:40:30,683] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 20:40:30,693] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:62349 )
  INFO root [2019-09-19 20:40:30,693] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:62349, SOMAXCONN=5.
  INFO root [2019-09-19 20:40:30,693] - server-configs:
  INFO root [2019-09-19 20:40:30,693] - 	internalTcpAddr : 192.168.202.54:62349
  INFO root [2019-09-19 20:40:30,693] - 	componentID : 6000
  WARN root [2019-09-19 20:40:30,694] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 20:40:32,199] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 664 ----
  INFO root [2019-09-19 20:40:32,199] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 20:40:32,200] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 20:40:32,200] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 20:40:32,600] - ---- cellappmgr is running ----
 DEBUG root [2019-09-19 20:40:32,600] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 20:40:39,741] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 20:40:39,818] - Components::findComponents: found logger, addr:192.168.202.54:62352
  INFO root [2019-09-19 20:40:39,818] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 20:40:39,818] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 20:40:39,820] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 20:40:39,922] - Components::findComponents: found dbmgr, addr:192.168.202.54:62353
  INFO root [2019-09-19 20:40:39,923] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 20:40:39,923] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 20:40:39,929] - Components::findComponents: found dbmgr, addr:192.168.202.54:62353
  WARN root [2019-09-19 20:40:40,929] - Forwarding logs to logger[192.168.202.54:62352/0/0/0]...
 ERROR root [2019-09-19 20:44:42,381] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=885292644095, lastReceivedTime=885176664269, inactivityExceptionPeriod=599499489)
  INFO root [2019-09-19 20:44:42,385] - The following logs sent to logger failed:
  WARN root [2019-09-19 20:44:42,385] - ==> [2019-09-19 20:44:42 380] TCPPacketReceiver::processPendingEvents(192.168.202.54:62352): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:44:42,386] - ==> [2019-09-19 20:44:42 382] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=885292644095, lastReceivedTime=885176664269, inactivityExceptionPeriod=599499489)
  WARN root [2019-09-19 20:44:42,477] - TCPPacketReceiver::processPendingEvents(192.168.202.54:62353): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:44:42,480] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=885293637394, lastReceivedTime=885176985059, inactivityExceptionPeriod=599499489)
  INFO root [2019-09-19 20:44:42,494] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 20:44:42,495] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 20:44:42,495] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 20:44:42,496] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 20:44:44,319] - 
  INFO root [2019-09-19 20:44:44,320] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 20:44:44,321] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 20:44:44,321] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 20:44:44,322] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 20:44:44,334] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:62437 )
  INFO root [2019-09-19 20:44:44,334] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:62437, SOMAXCONN=5.
  INFO root [2019-09-19 20:44:44,335] - server-configs:
  INFO root [2019-09-19 20:44:44,335] - 	internalTcpAddr : 192.168.202.54:62437
  INFO root [2019-09-19 20:44:44,335] - 	componentID : 6000
  WARN root [2019-09-19 20:44:44,337] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 20:44:44,806] - Components::findLogger: found logger, addr:192.168.202.54:62434
  INFO root [2019-09-19 20:44:44,806] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 20:44:44,806] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 20:44:44,812] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 7252 ----
  INFO root [2019-09-19 20:44:44,814] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 20:44:44,815] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 20:44:44,815] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 20:44:45,298] - ---- cellappmgr is running ----
 DEBUG root [2019-09-19 20:44:45,299] - Components::process(): Request for the process of identity...
  WARN root [2019-09-19 20:44:51,438] - Forwarding logs to logger[192.168.202.54:62434/0/0/0]...
  INFO root [2019-09-19 20:52:48,870] - EventDispatcher()::~EventDispatcher: Num timers = 2
  INFO root [2019-09-19 20:56:08,424] - 
  INFO root [2019-09-19 20:56:08,433] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 20:56:08,435] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 20:56:08,435] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 20:56:08,435] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 20:56:08,449] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:62687 )
  INFO root [2019-09-19 20:56:08,450] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:62687, SOMAXCONN=5.
  INFO root [2019-09-19 20:56:08,450] - server-configs:
  INFO root [2019-09-19 20:56:08,450] - 	internalTcpAddr : 192.168.202.54:62687
  INFO root [2019-09-19 20:56:08,450] - 	componentID : 6000
  WARN root [2019-09-19 20:56:08,451] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 20:56:09,956] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 9728 ----
  INFO root [2019-09-19 20:56:09,957] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 20:56:09,957] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 20:56:09,957] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 20:56:10,463] - ---- cellappmgr is running ----
 DEBUG root [2019-09-19 20:56:10,464] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 20:56:17,631] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 20:56:17,689] - Components::findComponents: found logger, addr:192.168.202.54:62686
  INFO root [2019-09-19 20:56:17,689] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 20:56:17,689] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 20:56:17,691] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 20:56:17,795] - Components::findComponents: found dbmgr, addr:192.168.202.54:62691
  INFO root [2019-09-19 20:56:17,795] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 20:56:17,795] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 20:56:17,802] - Components::findComponents: found dbmgr, addr:192.168.202.54:62691
  WARN root [2019-09-19 20:56:18,802] - Forwarding logs to logger[192.168.202.54:62686/0/0/0]...
 ERROR root [2019-09-19 20:58:35,549] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=893624324764, lastReceivedTime=893352846744, inactivityExceptionPeriod=599499873)
  INFO root [2019-09-19 20:58:35,555] - The following logs sent to logger failed:
  WARN root [2019-09-19 20:58:35,555] - ==> [2019-09-19 20:58:35 545] TCPPacketReceiver::processPendingEvents(192.168.202.54:62686): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:58:35,556] - ==> [2019-09-19 20:58:35 550] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=893624324764, lastReceivedTime=893352846744, inactivityExceptionPeriod=599499873)
  WARN root [2019-09-19 20:58:35,660] - TCPPacketReceiver::processPendingEvents(192.168.202.54:62691): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:58:35,667] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=893625501167, lastReceivedTime=893354443161, inactivityExceptionPeriod=599499873)
  INFO root [2019-09-19 20:58:35,686] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 20:58:35,698] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 20:58:35,698] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 20:58:35,699] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 21:07:18,273] - 
  INFO root [2019-09-19 21:07:18,274] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 21:07:18,275] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 21:07:18,275] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 21:07:18,275] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 21:07:18,285] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:62779 )
  INFO root [2019-09-19 21:07:18,286] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:62779, SOMAXCONN=5.
  INFO root [2019-09-19 21:07:18,286] - server-configs:
  INFO root [2019-09-19 21:07:18,286] - 	internalTcpAddr : 192.168.202.54:62779
  INFO root [2019-09-19 21:07:18,286] - 	componentID : 6000
  WARN root [2019-09-19 21:07:18,286] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 21:07:19,790] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 16948 ----
  INFO root [2019-09-19 21:07:19,791] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 21:07:19,791] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 21:07:19,792] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 21:07:20,185] - ---- cellappmgr is running ----
 DEBUG root [2019-09-19 21:07:20,185] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 21:07:27,313] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 21:07:27,333] - Components::findComponents: found logger, addr:192.168.202.54:62777
  INFO root [2019-09-19 21:07:27,333] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 21:07:27,334] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 21:07:27,335] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 21:07:27,335] - Components::findComponents: found logger, addr:192.168.202.54:62777
  INFO root [2019-09-19 21:07:27,339] - Components::findComponents: found dbmgr, addr:192.168.202.54:62780
  INFO root [2019-09-19 21:07:27,339] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 21:07:27,339] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 21:07:27,441] - Components::findComponents: found dbmgr, addr:192.168.202.54:62780
  WARN root [2019-09-19 21:07:28,442] - Forwarding logs to logger[192.168.202.54:62777/0/0/0]...
 ERROR root [2019-09-19 21:10:43,393] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=900902761934, lastReceivedTime=900650930711, inactivityExceptionPeriod=599527881)
  INFO root [2019-09-19 21:10:43,402] - The following logs sent to logger failed:
  WARN root [2019-09-19 21:10:43,402] - ==> [2019-09-19 21:10:43 389] TCPPacketReceiver::processPendingEvents(192.168.202.54:62777): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 21:10:43,403] - ==> [2019-09-19 21:10:43 393] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=900902761934, lastReceivedTime=900650930711, inactivityExceptionPeriod=599527881)
  WARN root [2019-09-19 21:10:43,495] - TCPPacketReceiver::processPendingEvents(192.168.202.54:62780): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 21:10:43,497] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=900903798165, lastReceivedTime=900652465609, inactivityExceptionPeriod=599527881)
  INFO root [2019-09-19 21:10:43,515] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 21:10:43,537] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 21:10:43,537] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 21:10:43,538] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 21:14:08,742] - 
  INFO root [2019-09-19 21:14:08,743] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 21:14:08,744] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 21:14:08,744] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 21:14:08,745] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 21:14:08,799] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:62901 )
  INFO root [2019-09-19 21:14:08,799] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:62901, SOMAXCONN=5.
  INFO root [2019-09-19 21:14:08,799] - server-configs:
  INFO root [2019-09-19 21:14:08,814] - 	internalTcpAddr : 192.168.202.54:62901
  INFO root [2019-09-19 21:14:08,815] - 	componentID : 6000
  WARN root [2019-09-19 21:14:08,839] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 21:14:10,347] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 8700 ----
  INFO root [2019-09-19 21:14:10,347] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 21:14:10,348] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 21:14:10,348] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 21:14:10,770] - ---- cellappmgr is running ----
 DEBUG root [2019-09-19 21:14:10,771] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 21:14:17,911] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 21:14:17,952] - Components::findComponents: found logger, addr:192.168.202.54:62898
  INFO root [2019-09-19 21:14:17,952] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 21:14:17,952] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 21:14:17,954] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 21:14:17,957] - Components::findComponents: found logger, addr:192.168.202.54:62898
  INFO root [2019-09-19 21:14:17,960] - Components::findComponents: found dbmgr, addr:192.168.202.54:62900
  INFO root [2019-09-19 21:14:17,960] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 21:14:17,960] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 21:14:18,060] - Components::findComponents: found dbmgr, addr:192.168.202.54:62900
  WARN root [2019-09-19 21:14:19,061] - Forwarding logs to logger[192.168.202.54:62898/0/0/0]...
 ERROR root [2019-09-19 21:38:36,823] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=917637071699, lastReceivedTime=917355532559, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-19 21:38:36,829] - The following logs sent to logger failed:
  WARN root [2019-09-19 21:38:36,829] - ==> [2019-09-19 21:38:36 821] TCPPacketReceiver::processPendingEvents(192.168.202.54:62898): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 21:38:36,830] - ==> [2019-09-19 21:38:36 824] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=917637071699, lastReceivedTime=917355532559, inactivityExceptionPeriod=599500000)
  WARN root [2019-09-19 21:38:36,921] - TCPPacketReceiver::processPendingEvents(192.168.202.54:62900): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 21:38:36,923] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=917638066353, lastReceivedTime=917356363402, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-19 21:38:36,932] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 21:38:36,938] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 21:38:36,938] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 21:38:36,938] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 21:38:38,616] - 
  INFO root [2019-09-19 21:38:38,640] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 21:38:38,641] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 21:38:38,641] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 21:38:38,641] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 21:38:38,668] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:63161 )
  INFO root [2019-09-19 21:38:38,669] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:63161, SOMAXCONN=5.
  INFO root [2019-09-19 21:38:38,669] - server-configs:
  INFO root [2019-09-19 21:38:38,669] - 	internalTcpAddr : 192.168.202.54:63161
  INFO root [2019-09-19 21:38:38,669] - 	componentID : 6000
  WARN root [2019-09-19 21:38:38,673] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 21:38:39,056] - Components::findLogger: found logger, addr:192.168.202.54:63157
  INFO root [2019-09-19 21:38:39,056] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 21:38:39,057] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 21:38:39,062] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 41181. PID: 18172 ----
  INFO root [2019-09-19 21:38:39,063] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 21:38:39,064] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 21:38:39,065] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 21:38:39,488] - ---- cellappmgr is running ----
 DEBUG root [2019-09-19 21:38:39,489] - Components::process(): Request for the process of identity...
  WARN root [2019-09-19 21:38:45,630] - Forwarding logs to logger[192.168.202.54:63157/0/0/0]...
 ERROR root [2019-09-19 21:41:16,094] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=919229782513, lastReceivedTime=919152956209, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-19 21:41:16,102] - The following logs sent to logger failed:
  WARN root [2019-09-19 21:41:16,103] - ==> [2019-09-19 21:41:16 094] TCPPacketReceiver::processPendingEvents(192.168.202.54:63157): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 21:41:16,103] - ==> [2019-09-19 21:41:16 095] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=919229782513, lastReceivedTime=919152956209, inactivityExceptionPeriod=599500000)
  WARN root [2019-09-19 21:41:16,187] - TCPPacketReceiver::processPendingEvents(192.168.202.54:63159): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 21:41:16,192] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=919230756243, lastReceivedTime=919153637263, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-19 21:41:16,209] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 21:41:16,213] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 21:41:16,213] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 21:41:16,214] - ThreadPool::destroy(): starting size 2.
